src/main/kotlin/org/rust/ide/utils/import/ImportCandidatesCollector2.kt

```diff
@@ -399,6 +397,7 @@ private fun ImportContext2.createPathWithImportAdded(usePath: String): RsPath? {
 
 private fun QualifiedNamedItem2.toImportInfo(defMap: CrateDefMap, modData: ModData, needExternCrate: Boolean): ImportInfo {
     val crateName = path.first()
+    val path = path.map2Array(String::escapeIdentifierIfNeeded)
```

Comment:
Looks like it's not enough to support case when crate name is keyword because you don't transform `crateName` when you pass it to `ExternCrateImportInfo`

https://user-images.githubusercontent.com/2539310/159128889-d076e64a-cf14-4c71-a470-d9ee28faf579.mov

Reply:
It would be great to add the corresponding test